ios/BetterSocial.xcodeproj/project.pbxproj

```diff
@@ -764,7 +764,7 @@
 					};
 					888DAA1429B48C8900311304 = {
 						DevelopmentTeam = J22M3M9VYU;
-						ProvisioningStyle = Automatic;
+						ProvisioningStyle = Manual;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Changing the `ProvisioningStyle` from "Automatic" to "Manual" can have implications on your CI/CD pipeline. If you're using a service like GitHub Actions or Jenkins, they might not be able to automatically sign the app for deployment. Make sure your CI/CD pipeline is configured to handle manual provisioning.
```diff
-    ProvisioningStyle = Manual;
+    ProvisioningStyle = Automatic;
```
If you intentionally changed this to "Manual", ensure that all developers in your team are aware of this change and have access to the necessary certificates and provisioning profiles to build the project locally.


<!-- This is an auto-generated comment by OSS CodeRabbit -->